Setup/UpgradeSchema.php

```diff
@@ -411,5 +411,11 @@ protected function createOptimizationIndexes(SchemaSetupInterface $installer)
             $installer->getIdxName('sales_payment_transaction', ['txn_id']),
             ['txn_id']
         );
+
+        $installer->getConnection()->addIndex(
+            $installer->getTable('buckaroo_magento2_second_chance'),
+            $installer->getIdxName('buckaroo_magento2_second_chance', ['store_id']),
```

Comment:
Hello, for new indexes we use https://github.com/buckaroo-it/Magento2/blob/develop/etc/db_schema.xml#L48 , will be great if you use as well

Reply:
Hi there, thanks for the info, I'll do the changes.